src/Init/Data/UInt/Lemmas.lean

```diff
@@ -133,6 +133,9 @@ declare_uint_theorems UInt32
 declare_uint_theorems UInt64
 declare_uint_theorems USize
 
+theorem USize.toNat_ofNat_of_lt32 {n : Nat} (h : n < 4294967296) : toNat (ofNat n) = n :=
```

Comment:
```suggestion
theorem USize.toNat_ofNat_of_lt_32 {n : Nat} (h : n < 4294967296) : toNat (ofNat n) = n :=
```
Just on the theory that we write e.g. `_lt_two`?

Reply:
@kim-em My inspiration for the current naming was `ofNat32`. My opinion on this naming is that the `_32` suggests to me a separate `32` term whereas `lt32` suggests a variation on the `lt` . The later feels more natural to what the theorem is doing. The former would fit more to me  if it was `lt_two_pow_32` but that is way too long. Regardless, I do not feel that strongly either way.